src/sentry/static/sentry/app/views/issueList/overview.tsx

```diff
@@ -53,49 +62,88 @@ const DEFAULT_GRAPH_STATS_PERIOD = '24h';
 const STATS_PERIODS = new Set(['14d', '24h']);
 const DYNAMIC_COUNTS_STATS_PERIODS = new Set(['14d', '24h', 'auto']);
 
-const IssueListOverview = createReactClass({
-  displayName: 'IssueListOverview',
-
-  propTypes: {
+type Params = {
+  orgId: string;
+};
+
+type Props = {
+  api: Client;
+  location: Location;
+  organization: Organization;
+  params: Params;
+  selection: GlobalSelection;
+  savedSearch: SavedSearch;
+  savedSearches: SavedSearch[];
+  savedSearchLoading: boolean;
+  tags: TagCollection;
+} & RouteComponentProps<{searchId?: string}, {}>;
+
+type State = {
+  groupIds: string[];
+  selectAllActive: boolean;
+  realtimeActive: boolean;
+  pageLinks: string;
+  queryCount: number;
+  queryMaxCount: number;
+  error: string | null;
+  isSidebarVisible: boolean;
+  issuesLoading: boolean;
+  tagsLoading: boolean;
+  memberList: ReturnType<typeof indexMembersByProject>;
+  query?: string;
+};
+
+type EndpointParams = Partial<GlobalSelection['datetime']> & {
+  project: number[];
+  environment: string[];
+  query?: string;
+  sort?: string;
+  statsPeriod?: string;
+  groupStatsPeriod?: string;
+  cursor?: string;
+  page?: number | string;
+};
+
+class IssueListOverview extends React.Component<Props, State> {
+  static propTypes: any = {
```

Comment:
You could remove the propTypes as this is a view and won't really benefit from proptype checks much, as IssueListContainer is already in typescript.